scrapy/downloadermiddlewares/chunked.py

```diff
@@ -1,21 +0,0 @@
-import warnings
-
-from scrapy.exceptions import ScrapyDeprecationWarning
-from scrapy.utils.http import decode_chunked_transfer
```

Comment:
what about this function?

Reply:
Ah, I see, it was only deprecated in 1.7. Makes sense to keep it for now.

Reply:
Yes, I removed it in an initial version of this branch but then I realized it was deprecated very recently (#3859)